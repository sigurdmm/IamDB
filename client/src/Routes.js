import { Switch, Route } from 'react-router-dom';

// eslint-disable-next-line import/no-named-as-default
import HomePage from './pages/HomePage';

// eslint-disable-next-line import/no-named-as-default
import FilmPage from './pages/FilmPage';

// eslint-disable-next-line import/no-named-as-default
import ActorPage from './pages/ActorPage';

const Routes = () => <Switch>
  <Route exact path="/" component={HomePage}/>
  <Route exact path="/media/:id" component={FilmPage}/>
  <Route exact path="/actor/:id" component={ActorPage}/>
</Switch>;
export default Routes;
