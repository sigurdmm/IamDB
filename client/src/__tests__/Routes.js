import { shallow } from 'enzyme';

import Routes from '../Routes';

describe('<Router/>', () => {
  it('renders properly', () => {
    const wrapper = shallow(<Routes/>);

    expect(wrapper).toMatchSnapshot();
  });
});
