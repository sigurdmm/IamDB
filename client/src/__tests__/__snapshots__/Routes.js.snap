// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Router/> renders properly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/media/:id"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/actor/:id"
  />
  <Redirect
    exact={true}
    push={false}
    to="/"
  />
</Switch>
`;
